<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>903</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageCorrelation</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Ref. Pos. 2</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>In-plane Rotation Angle</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dsb_rotAngle">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <double>-180.000000000000000</double>
              </property>
              <property name="maximum">
               <double>180.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Pixels</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_ref2_pxls"/>
              </item>
             </layout>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>XMotor Pos</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="dsb_ref2_x">
                  <property name="suffix">
                   <string> um</string>
                  </property>
                  <property name="decimals">
                   <number>0</number>
                  </property>
                  <property name="minimum">
                   <double>-10000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>10000.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>YMotor Pos</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="dsb_ref2_y">
                  <property name="suffix">
                   <string>  um</string>
                  </property>
                  <property name="decimals">
                   <number>0</number>
                  </property>
                  <property name="minimum">
                   <double>-10000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>10000.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>0.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pb_grabXY_2">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(85, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Insert Current Pos</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pb_apply_calculation">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pb_export_param">
           <property name="text">
            <string>Export Parameters</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_import_param">
           <property name="text">
            <string>Import Parameters</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Ref. Pos. 1</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Pixels</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_ref1_pxls"/>
              </item>
             </layout>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>XMotor Pos</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="dsb_ref1_x">
                  <property name="suffix">
                   <string> um</string>
                  </property>
                  <property name="decimals">
                   <number>0</number>
                  </property>
                  <property name="minimum">
                   <double>-10000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>10000.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>YMotor Pos</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="dsb_ref1_y">
                  <property name="suffix">
                   <string> um</string>
                  </property>
                  <property name="decimals">
                   <number>0</number>
                  </property>
                  <property name="minimum">
                   <double>-10000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>10000.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pb_grabXY_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(85, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Insert Current Pos</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_scale_info">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 255);</string>
         </property>
         <property name="text">
          <string>Scaling Info</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Selection Mode</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <widget class="QRadioButton" name="rb_nav_mode">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Navigation</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="rb_calib_mode">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Calibration</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Edit Images</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Opacity</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QSlider" name="hsb_ref_img1_op">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="hsb_ref_img1_op_num">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Image 1</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QSlider" name="hsb_ref_img2_op">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="sliderPosition">
                <number>70</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="hsb_ref_img2_op_num">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Image 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Translation</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>231</width>
             <height>127</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Nikon Coordinates: </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Origin:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>X (um)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QSpinBox" name="spinBox"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Y (um) </string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QSpinBox" name="spinBox_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Calculated Motor Positions</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>XMotor Pos</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dsb_calc_y">
              <property name="suffix">
               <string> um</string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>-10000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10000.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>YMotor Pos</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dsb_calc_x">
              <property name="suffix">
               <string> um</string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>-10000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pb_gotoTargetPos">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>GO</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Reference Image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="GraphicsLayoutWidget" name="ref_view">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>Transformed</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="GraphicsLayoutWidget" name="labaxis_view">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_refImage"/>
    <addaction name="actionAdd_refImage2"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save XANES Stack</string>
   </property>
  </action>
  <action name="actionSave_labImage">
   <property name="text">
    <string>Save Lab Axis Image</string>
   </property>
  </action>
  <action name="actionSave_Spectrum">
   <property name="text">
    <string>Save Spectrum</string>
   </property>
  </action>
  <action name="actionLoad_Energy_List">
   <property name="text">
    <string>Load Energy List</string>
   </property>
  </action>
  <action name="actionLoad_XRF_Map">
   <property name="text">
    <string>Load XRF Map</string>
   </property>
  </action>
  <action name="actionLoad_refImage">
   <property name="text">
    <string>Load Ref Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAdd_refImage2">
   <property name="text">
    <string>Load Ref Image2</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
